test/limits/mzcompose.py

```diff
@@ -1305,7 +1305,9 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
                     WORKERS {args.workers}
                 )
             )
-        """
+        """,
```

Comment:
This is not needed I think, with the correct `MZ_BOOTSTRAP_ROLE` setting this query should be runnable as user `materialize` on the default port.